backend/tracim_backend/tests/functional/test_message.py

```diff
@@ -20,9 +20,10 @@ def create_events_and_messages(session) -> tracim_event.Event:
             fields={"example": "hello"},
         )
         session.add(event)
-        messages.append(
-            tracim_event.Message(event=event, receiver_id=1, read=datetime.datetime.utcnow())
-        )
+        read_datetime = datetime.datetime.utcnow()
+        if unread:
+            read_datetime = None
```

Comment:
Not really relevant here as it is a test method, but in general it is easier and more powerful to give `read_datetime` as argument to the method instead of only a boolean.